src/classes/ADDR_Seasonal_UTIL.cls

```diff
@@ -210,16 +211,17 @@ public with sharing class ADDR_Seasonal_UTIL {
             }
         }
 
-        // now check for overlaps within each household's saved addresses
-        list<Id> listAccId = new list<Id>();
-        for (Address__c addr : listAddrS) {
-            listAccId.add(addr.Parent_Account__c);
-        }        
-        map<Id, list<Address__c>> mapAccIdListAddrs = ADDR_Addresses_UTIL.getMapParentIdListAddrs(listAccId);
+        // now check for overlaps within each parent's saved addresses      
+        Map<Id, List<Address__c>> mapAccIdListAddrs = ADDR_Addresses_UTIL.getParentIdToAddrs(listAddrS);
```

Comment:
Calling the new method. The logic wasn't right here, because the parameter did not include parent Contact IDs.